tests/notifications/test-class-yoast-notification-center-double.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * @package Yoast\Tests\Notifications
+ */
+
+/**
+ * Class Test_Yoast_Notification_Center
+ */
+class Yoast_Notification_Center_Double extends Yoast_Notification_Center {
```

Comment:
This file isn't a test class, why prefixing this with 'test-' ? https://github.com/Yoast/wordpress-seo/blob/trunk/phpunit.xml#L19 -> seeing this, phpunit probably might see this file as a test file.